routes/user/setting.go

```diff
@@ -607,7 +607,12 @@ func SettingsApplicationsPost(c *context.Context, f form.NewAccessToken) {
 		Name: f.Name,
 	}
 	if err := models.NewAccessToken(t); err != nil {
-		c.ServerError("NewAccessToken", err)
+		if models.IsErrAccessTokenNameAlreadyExist(err) {
+			c.Flash.Error("NewAccessToken: " + err.Error())
```

Comment:
We should add an i18n entry in the locale file `locale_en-US.ini`, e.g. `settings.token_name_exists`, and use `c.Tr(...)` here. (see an example on line 619)

Reply:
hm, do we need to run an extra command to get the i18n entry?

the error message is `token_name_exists` when running gogs with `make web`